linkup-cli/src/start.rs

```diff
@@ -114,3 +117,16 @@ pub fn save_state(state: LocalState) -> Result<(), CliError> {
 
     Ok(())
 }
+
+fn check_local_not_started() -> Result<(), CliError> {
+    let state = get_state()?;
+    for service in state.services {
```

Comment:
**_suggestion/nit:_**
I guess you don't really need a copy if this object here, so a reference should be enough.
```suggestion
    for service in &state.services {
```